kube_controller_manager/README.md

```diff
@@ -1,55 +1,57 @@
 # Agent Check: Kubernetes Controller Manager
 
+![Kube Controller Manager dashboard][1]
+
 ## Overview
 
-This check monitors the [Kubernetes Controller Manager][1], part of the Kubernetes control plane.
+This check monitors the [Kubernetes Controller Manager][2], part of the Kubernetes control plane.
 
 **Note**: This check does not collect data for Amazon EKS clusters, as those services are not exposed.
 
 ## Setup
 
 ### Installation
 
-The Kubernetes Controller Manager check is included in the [Datadog Agent][2] package, so you do not
+The Kubernetes Controller Manager check is included in the [Datadog Agent][3] package, so you do not
 need to install anything else on your server.
 
 ### Configuration
 
 This integration requires access to the controller manager's metric endpoint. It is usually not
```

Comment:
We can also document the fact that to metrics endpoint access is usually configurable thanks to RBAC which are automatically created for the agent went it is deployed with the helm chart our the operator.

Reply:
not sure what you mean by this, can you clarify, please?

Reply:
it is just that the initial sentences was not clear about what it means to have "access" to the metrics endpoint.

in practice is means:

* have access to the IP/Port of the controller-manager process
* On "recent" kubernetes clusters, have the `/metrics` GET rbac, like in the [helm chart](https://github.com/DataDog/helm-charts/blob/58bf52e4e342c79dbec95659458f7de8c5de7e6c/charts/datadog/templates/rbac.yaml#L66-L69)

```yaml
- nonResourceURLs:
  - "/metrics"
  verbs:
  - get
```

Reply:
OK! I will add that to the README